### DISCUSSION TITLE
Failed building

### DISCUSSION THREAD

**Discussion by semilore8** [html2app/discussions#8] [General]
- the build fail when ever I use any latest latest plugin, like when I wanted to use capacitor/app 5.0.6 it failed but when I downgrade to 4 as it is in your github plug in example it build. That my first problem
- the next one is that it does not use to catch error that make you know what's wrong it only used to log undefined/undefined. That my second 
-  the 3rd one is that you should have a doc about the configuration file options and vals.
....

**Comment by yandeu** [comment]
- I believe I haven't upgraded to Capacitor@5 yet.
- True, error message/logs are still hidden from the clients.
- I agree, for the service to be easy to use, it needs better docs.

To do all of this, I simply need more time, which I haven't at the moment :(

**Reply by semilore8** [reply]
Ok alert. When u done with some or all

**Comment by semilore8** [comment]
When using more that one plugin like Capacitor app4 and toast it doesn't build it  failed. But when I use one at in the configuration it build

**Reply by yandeu** [reply]
Interesting. I just opened a new issue for it.
